frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx

```diff
@@ -9,34 +9,88 @@ import {
     Tooltip,
     Legend,
     TimeScale,
+    Chart,
 } from 'chart.js';
 import { Line } from 'react-chartjs-2';
 import 'chartjs-adapter-date-fns';
-import { Paper, Theme, useTheme } from '@mui/material';
+import { Paper, Theme, Typography, useTheme } from '@mui/material';
 import {
     useLocationSettings,
     type ILocationSettings,
 } from 'hooks/useLocationSettings';
-import { formatDateYMD } from 'utils/formatDate';
 import { ExecutiveSummarySchema } from 'openapi';
+import { ChartTooltip, TooltipState } from './ChartTooltip/ChartTooltip';
 
-const createOptions = (theme: Theme, locationSettings: ILocationSettings) =>
+const createOptions = (
+    theme: Theme,
+    locationSettings: ILocationSettings,
+    setTooltip: React.Dispatch<React.SetStateAction<TooltipState | null>>,
+) =>
     ({
         responsive: true,
         plugins: {
             legend: {
                 position: 'bottom',
+                labels: {
+                    boxWidth: 12,
+                    padding: 30,
+                    // usePointStyle: true,
+                    generateLabels: (chart: Chart) => {
+                        const datasets = chart.data.datasets;
+                        const {
+                            labels: {
+                                usePointStyle,
+                                pointStyle,
+                                textAlign,
+                                color,
+                            },
+                        } = chart?.legend?.options || {
+                            labels: {},
+                        };
+                        return (chart as any)
```

Comment:
I have some issues with types on Chart.js. Hopefully I'll be able to fix it in next iteration